src/main/java/seedu/address/model/interview/Interview.java

```diff
@@ -61,6 +62,15 @@ public boolean isInterviewForPosition(Position p) {
         return position.isSamePosition(p);
     }
 
+    /**
+     * Checks if the given interview will conflict with the current interview.
+     */
+    public boolean isConflictingInterview(Interview i) {
+        return i.isInterviewForApplicant(this.applicant)
+                && !(i.date.isBefore(this.date.minus(59, ChronoUnit.MINUTES))
+                        || i.date.isAfter(this.date.plusMinutes(59)));
```

Comment:
I understand the logic here but maybe can split into a few lines for better readability? It may not be very intuitive for a first time reader to understand